MTGOSDK/src/API/Interface/WindowUtilities.cs

```diff
@@ -38,6 +38,9 @@ public static class WindowUtilities
   /// </exception>
   public static ICollection<dynamic> GetWindows()
   {
+#if NETSTANDARD2_0
+    throw new PlatformNotSupportedException();
+#else
```

Comment:
This is due to a (possibly?) unresolved assembly reference to PresentationFramework when targeting the .NET Standard 2.0 TFM, despite using the `<UseWPF>` property.